Source/ORTS.sln

```diff
@@ -55,115 +55,184 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ContentChecker", "ContentCh
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|Mixed Platforms = Debug|Mixed Platforms
+		Release|Any CPU = Release|Any CPU
```

Comment:
this is generated code/config, and will be changed/cleaned up in subsequent PR anyway

Reply:
I've read through this more carefully, and there's only one difference between the existing configurations and the new ones: RunActivity is built as Any CPU. Since this won't actually work at present, due to RailDriver, please remove this new configuration.